src/plugins/toast/index.js

```diff
@@ -27,9 +27,9 @@ const plugin = {
         // destroy watcher
         watcher && watcher()
         if (typeof options === 'string') {
-          $vm.text = options
+          mergeOptions($vm, Object.assign({}, pluginOptions, {text: options}))
```

Comment:
use `object-assign` instead of `Object.assign` to avoid compatibility issue on Android.

``` js
import objectAssign from 'object-assign'
```